lib/main.dart

```diff
@@ -283,9 +283,6 @@ class Finamp extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    const Color accentColor = Color(0xFF00A4DC);
```

Comment:
I'd like to keep the original accent color here for reference :)

Reply:
Added it back in `lib/color_schemes.g.dart` with a comment.